src/settings-ui/Settings.UI/SettingsXAML/Controls/ColorFormatEditor.xaml

```diff
@@ -36,11 +36,11 @@
             </Grid>
         </DataTemplate>
 
-        <DataTemplate x:Key="ColorParameterTemplate" x:DataType="local:ColorFormatParameter">
+        <DataTemplate x:Key="ColorParameterTemplate" x:DataType="custom:ColorFormatParameter">
```

Comment:
Maybe increase the spacing between the sections a bit.. either by increasing the spacing (red) or decrease (blue) the bottom part of each section
![image](https://github.com/microsoft/PowerToys/assets/9866362/95c63aee-c6d8-4ef3-beba-ce8ef767eb3a)

Reply:
I knew you would mention this. 😄 My choice is based on...
![image](https://github.com/microsoft/PowerToys/assets/65828559/5fe00601-8a12-4fb8-8090-560b9ac80b57)

> Using Title, Subtitle and Body with 12epx spacing.

https://learn.microsoft.com/nl-nl/windows/apps/design/basics/content-basics#text--hierarchy

Reply:
I think the spacing is fine, but there is no hierarchy anymore. The title of a 'section' should provide more spacing so you know that the title/description belongs to the legend.

Reply:
Maybe a Style would be better, then

Reply:
Just adding an additional top-margin to the titles/descriptions should work!

Reply:
Sorry, your suggestion contradicts the guidance of Microsoft. It's confusing me and I'm not sure what to do, but my preference would be to follow the guidance.

Reply:
> so you know that the title/description belongs to the legend.

I think the colon handles that.

Reply:
> Sorry, your suggestion contradicts the guidance of Microsoft. It's confusing me and I'm not sure what to do, but my preference would be to follow the guidance.

The information shown in this pop-up are sections, right? There needs to be more vertical spacing between sections vs. the title/label and content of a section. This helps to create a visual hierarchy. I don't believe the _guidelines_ you mentioned describe the spacing between sections?

Here's an example from the Fluent Windows Visual Library in Figma, which should be considered as our redlines for things. See how the spacing between the different sections is slightly higher (red) than the spacing between the header of the textbox (blue)?
<img width="604" alt="image" src="https://github.com/microsoft/PowerToys/assets/9866362/1aef6693-5e13-4c31-b5a2-ccbf2712e0a5">

Reply:
According to the design library, the spacing between sections (red) is 24px and spacing between header/child (blue) is 12px. Might be slightly different in XAML with the rendering of the TextBlock

Reply:
Okay, let's take one step back: Do we consider the legend as a **control**, or as **text**? That's a big difference. [MS Learn](https://learn.microsoft.com/nl-nl/windows/apps/design/basics/content-basics#spacing-and-gutters) actually says to use **8px** between control and _its_ header, but looks clear on the 12px between text.
Also: I don't know Figma. If anything on MS Learn is outdated, that info should be updated. Otherwise I'm sticking to that.

Reply:
The docs are outdated here and there (as you can see your article is from 2021) - I think for this topic specifically, they are still correct.

In Figma it's 12px, but could very well be it's 8px in **e**px. It's a detail, the point is that the current spacing between the headers of the sections **does not provide the correct spacing**. 

For redlines and inspiration, the Windows Design team recommends to use the Fluent Windows Design Library - it's listed on the `Toolkits and libraries` page:
![image](https://github.com/microsoft/PowerToys/assets/9866362/8d031ff2-fa5b-43fd-bf0f-8596d4bc67c2)

Reply:
Looking at this, I wonder if we can make these titles less verbose?

"The following parameters:" => "**Parameters**"

"The red, green, blue and alpha values can be formatted to the following formats" => **"Formatting parameters"**

"Example: %ReX means red value in hex uppercase two digits format" => "Example: %ReX = ff"


Probably there are better ways to describe these :)

Reply:
I've reset the visual to what it was.

Reply:
I made it 12 and 8.